contracts/modules/claims/Claims.sol

```diff
@@ -80,14 +80,18 @@ contract Claims is IClaims, LegacyMasterAware {
    * @param coverId Cover Id.
    */
   function submitClaim(uint coverId) external {
-    _submitClaim(coverId, msg.sender);
+    _submitClaim(coverId, msg.sender, 0);
```

Comment:
let's pass this as full cover amount here, as you're also returning it in `TC.coverInfo()` and it gets a bit confusing imo

Reply:
I would expect the default is to claim full amount, unless we make 0 signify full amount, because claiming 0 means nothing

Reply:
zero was a convention to indicate full amount. will change to full value.

Reply:
I think either way is fine as long as it's documented since 0 doesn't make sense for a claim

Reply:
Ended up just documenting it because the logic for setting it to a full amount implied changing the `Incidents` contract or adding a new dependant contract (`QuotationData`) to the `TokenController` contract, plus creating a new mock for `QuotationData` for `TokenController` tests, so it's not worth it.

Reply:
sounds good